src/detect-pangram.js

```diff
@@ -15,7 +15,14 @@
  * @return {boolean}
  */
 function detectPangram(phrase) {
-  // write code here
+  const alphabet = 'abcdefghijklmnopqrstuvwxyz';
+  const lowerCaseString = phrase.toLowerCase();
```

Comment:
имел в виду, что по названию нужно понять что в ней хранится, не логический тип, а содержимое
```suggestion
  const phraseLowerCase = phrase.toLowerCase();
```